src/Operators/spacings_and_areas_and_volumes.jl

```diff
@@ -138,3 +140,35 @@ The operators in this file fall into three categories:
 @inline Azᶠᶠᵃ(i, j, k, grid::RegularLatitudeLongitudeGrid) = @inbounds grid.radius^2 * deg2rad(grid.Δλ) * (hack_sind(grid.ϕᵃᶜᵃ[j])   - hack_sind(grid.ϕᵃᶜᵃ[j-1]))
 @inline Azᶠᶜᵃ(i, j, k, grid::RegularLatitudeLongitudeGrid) = Azᶜᶜᵃ(i, j, k, grid)
 @inline Azᶜᶠᵃ(i, j, k, grid::RegularLatitudeLongitudeGrid) = Azᶠᶠᵃ(i, j, k, grid)
+
+#####
+##### Generic functions for specified locations
+#####
+##### For example, Δx(i, j, k, Face, Center, LZ) is equivalent to = Δxᶠᶜᵃ(i, j, k, grid).
+#####
+##### We also use the function "volume" rather than `V`.
+#####
+
+location_code_xy(LX, LY) = Symbol(interpolation_code(LX), interpolation_code(LY), :ᵃ)
+location_code(LX, LY, LZ) = Symbol(interpolation_code(LX), interpolation_code(LY), interpolation_code(LZ))
+
+for LX in (:Center, :Face)
```

Comment:
In terms of style, do we tend to join double for loop into one line?  Not needed but I have seen that before and wanted to check to see if this is considered good practice, or not

Reply:
I think it's nice to do that actually --- just didn't think of it here.